corehq/apps/reports/util.py

```diff
@@ -690,6 +681,7 @@ def import_tableau_users(domain, web_user_specs):
         session = TableauAPISession.create_session_for_domain(domain)
     except TableauAPIError as e:
         _notify_tableau_exception(e, domain)
+        return
```

Comment:
What's different about this try/except block versus the others?  Should it `raise` the swallowed exception instead of `return`?

Reply:
Yeah that sounds logical to me - a partial success can be tricky to retry, and if there's a quick workaround like you suggest, then that seems like the best of both worlds